src/Installation/wine.md

```diff
@@ -0,0 +1,18 @@
+# Installing wine
+
+A wine version higher than 7 is required, 8.3+ is recommended.
```

Comment:
Provide a reasoning on why 8.3 is 'reccomended'

Reply:
Low Fragmentation Heap, without it heavier games lag

Reply:
ON the wiki Oak.

Reply:
https://www.winehq.org/news/2023030301
Low Fragmentation Heap got implemented, if youve played a bit heavier games on wine without LFH you would know.